.github/workflows/ci.yml

```diff
@@ -80,6 +80,11 @@ jobs:
           cd flink-kubernetes-webhook
           mvn verify -Dit.skip=false
           cd ..
+      - name: Tests in flink-kubernetes-docs
```

Comment:
We already have `mvn clean install -DskipTests -Pgenerate-docs` in ci.yml to guard the docs are re-generated accordingly. What I mean is to simply add `-Dgenerate-config-docs` here.
Maybe `-Pgenerate-config-docs` looks better.

Reply:
Could we simply reuse the existing profile?

Reply:
It also makes sense to reuse the profile. It seems we do not need to differentiate generating CRD docs and config docs.